src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Mapping/ToViewModel/ReturnAndAatfToSentOnRemoveSiteViewModelMapTests.cs

```diff
@@ -67,8 +67,51 @@ public void Map_GivenValidSource_PropertiesShouldBeMapped()
             result.SiteAddress.Should().Be(siteAddress);
             result.OperatorAddress.Should().Be(operatorAddress);
             result.WeeeSentOn.Should().BeEquivalentTo(weeeSentOn);
-            result.TonnageB2B.Should().Be(decimal.Parse(obligatedTonnage.B2B));
-            result.TonnageB2C.Should().Be(decimal.Parse(obligatedTonnage.B2C));
+            result.TonnageB2B.Should().Be(obligatedTonnage.B2B);
+            result.TonnageB2C.Should().Be(obligatedTonnage.B2C);
+        }
+
+        [Fact]
+        public void Map_GivenNoTonnages_TonnagesShouldBeSetToZero()
```

Comment:
Is the description of the test correct, looks more like testing the overall view model properties